nservicebus/pipeline/satellites.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating Satellites
+summary: Extension point for handling messages
+tags: 
+related:
+ - nservicebus/pipeline/customizing-v6
+ - nservicebus/pipeline/features
+redirects:
+---
+
+In NServiceBus Versions 5 and below, Satellites were extension points to handle messages that arrive in different queues other than the endpoint's main queue. This extension point came in handy when the robustness of a separate queue was needed without having to create and setup a brand new Endpoint and configure any needed message mappings.
```

Comment:
I, for one, welcome this new structure...seems to make it more of a step by step, appropriate as it is effectively migration doco.